lms/templates/ccx/schedule.html

```diff
@@ -29,80 +29,88 @@
   <div id="new-ccx-schedule"></div>
 </div>
 
-<section id="enter-date-modal" class="modal" aria-hidden="true">
-  <div class="inner-wrapper" role="dialog">
-    <button class="close-modal">
-      <i class="icon fa fa-remove"></i>
+<section id="enter-date-modal" class="modal"
+         tabindex="1" role="dialog" aria-labelledby="ccx_schedule_set_date_heading">
+  <div class="inner-wrapper">
+    <button class="close-modal" tabindex="2">
+      <i class="icon fa fa-remove" aria-hidden="true"></i>
       <span class="sr">
         ${_("Close")}
       </span>
     </button>
     <header>
-      <h2></h2>
+      <h2 tabindex="2" id="ccx_schedule_set_date_heading"></h2>
```

Comment:
i don't think you actually want to make this heading focusable.  Headings are not natively focusable.  Screen reader users will use an arrow key to consume non-focusable content.  remove ```tabindex```